packages/govuk-frontend/src/govuk/components/file-upload/file-upload.mjs

```diff
@@ -33,6 +33,15 @@ export class FileUpload extends ConfigurableComponent {
   /** @private */
   id
 
+  /** @private */
+  $announcements
+
+  /**
+   * @private
+   * @type {boolean | undefined}
```

Comment:
The `| undefined` is due to the field only being set for the first time inside the event listener for `dragenter` in the constructor, so it remains `undefined` until the first event is dispatched (which may never happen if the user never drags anything on the page).